Controllers/PizzaController.cs

```diff
@@ -0,0 +1,114 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Microsoft.AspNetCore.Mvc;
+using ContosoPizza.Models;
+using ContosoPizza.Services;
+
+namespace ContosoPizza.Controllers
+{
+    /// <summary>
+    /// Pizza Controller.
+    /// </summary>
+    /// <seealso cref="Microsoft.AspNetCore.Mvc.ControllerBase" />
+    [ApiController]
+    [Route("[controller]")]
+    public class PizzaController : ControllerBase
+    {
+        /// <summary>
+        /// Initializes a new instance of the <see cref="PizzaController"/> class.
+        /// </summary>
+        public PizzaController()
+        {
+        }
+
+        /// <summary>
+        /// Gets all pizzas.
+        /// </summary>
+        /// <returns></returns>
+        [HttpGet]
+        public ActionResult<List<Pizza>> GetAll() =>
+            PizzaService.GetAll();
+
+        /// <summary>
+        /// Gets the specified pizza by identifier.
+        /// </summary>
+        /// <param name="id">The identifier.</param>
+        /// <returns></returns>
+        [HttpGet("{id}")]
+        public ActionResult<Pizza> Get(int id)
+        {
+            var pizza = PizzaService.Get(id);
+
+            if (pizza == null)
+                return null;
+
+            return pizza;
+        }
+
+        /// <summary>
+        /// Creates the specified pizza.
+        /// </summary>
+        /// <param name="pizza">The pizza.</param>
+        /// <returns></returns>
+        [HttpPost]
+        public IActionResult Create(Pizza pizza)
+        {
+            PizzaService.Add(pizza);
+            return CreatedAtAction(nameof(Create), new { id = pizza.Id }, pizza);
+        }
+
+        /// <summary>
+        /// Updates the specified pizza by identifier.
+        /// </summary>
+        /// <param name="id">The identifier.</param>
+        /// <param name="pizza">The pizza.</param>
+        /// <returns></returns>
+        [HttpPost("{id}")]
+        public IActionResult Update(int id, Pizza pizza)
+        {
+            if (id != pizza.Id)
```

Comment:
Can we reframe this one, this condition is ambiguous as we would not know from id whether its pizza id or not